versioned/storage/store/src/main/java/org/projectnessie/versioned/storage/versionstore/CommitImpl.java

```diff
@@ -412,6 +417,6 @@ private void commitAddPut(
 
   private String contentIdFromContent(@Nonnull @jakarta.annotation.Nonnull ObjId contentValueId)
       throws ObjNotFoundException {
-    return new ContentMapping(persist).fetchContent(contentValueId).getId();
+    return contentMapping.fetchContent(contentValueId).getId();
```

Comment:
Can you do me a favor and open an issue indicating that we should pre-load these objects, so they are bulk-loaded and in turn available via the cache here?

Reply:
Here you are: https://github.com/projectnessie/nessie/issues/6673.